app/api/dao/user.py

```diff
@@ -3,7 +3,8 @@
 from app.api.email_utils import confirm_token
 from app.database.models.user import UserModel
 from app.utils.validation_utils import is_email_valid
-
+from app.utils import aws_upload
+import os
```

Comment:
incorrect import order

Reply:
@ramitsawhney27 Does it mean shift `import os` to top ?

Reply:
I think yes. Refer https://www.python.org/dev/peps/pep-0008/#imports and https://stackoverflow.com/questions/20762662/whats-the-correct-way-to-sort-python-import-x-and-from-x-import-y-statement

Reply:
@agarwalrounak Thanks for the link :)